jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/ResourceService.java

```diff
@@ -160,6 +213,15 @@ public void doGet(Request request, Response response, Callback callback, HttpCon
 
         try
         {
+            if (isStylesheetRequest(pathInContext))
+            {
+                if (sendStylesheet(pathInContext, request, response, callback))
```

Comment:
why is this a boolean return?  Do we expect failure to be frequent? Can't it just throw and let normal exception handling deal with it?